src/main.css

```diff
@@ -0,0 +1,25 @@
+h1 {
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-size: 20pt;
+    text-align: center
+}
+
+nav {
+    text-align: center;
+}
+
+#logo-main {
+    width: 900px;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+}
+
+body {
+    background: url(/assets/20180821-\ Hoh\ Rainforest48e-3.jpg);
+    background-size: 1700px 980px;
+}
```

Comment:
if you add 
background-repeat: no-repeat;

your background image wont repeat like it does now